conans/client/conf/__init__.py

```diff
@@ -684,3 +688,22 @@ def get_log_level_by_name(level_name):
             "notset": logging.NOTSET
         }
         return levels.get(str(level_name).lower())
+
+    @property
+    def config_install_interval(self):
+        try:
+            interval = self.get_item("general.config_install_interval")
+        except ConanException:
+            return None
+
+        match = re.search(r"(\d+)([mhd])", interval)
+        try:
+            value, unit = match.group(1), match.group(2)
+            if unit == 'm':
+                return timedelta(minutes=float(value))
+            elif unit == 'h':
+                return timedelta(hours=float(value))
+            else:
+                return timedelta(days=float(value))
+        except Exception as e:
+            raise ConanException("Incorrect definition of config_install_interval: %s" % interval)
```

Comment:
```suggestion
            raise ConanException("Incorrect definition of general.config_install_interval: %s" % interval)
```

it is in `[general]` section, isn't it?

Reply:
yes, I'll fix it, together with the test.